src/alignment.js

```diff
@@ -85,8 +85,35 @@ var alignWithDOM = function(nodeName, key, statics) {
 };
 
 
+/**
+ * Clears out any unvisited Nodes, as the corresponding virtual element
+ * functions were never called for them.
+ * @param {!Element} node
+ */
+var clearUnvisitedDOM = function(node) {
+  var data = getData(node);
+  var lastChild = node.lastChild;
```

Comment:
I [disagree](http://jsperf.com/cached-lastchild), but why ever do an extra lookup if you can avoid it with two simple lines?